cmd/agentctl/commands/kvdb.go

```diff
@@ -0,0 +1,200 @@
+//  Copyright (c) 2019 Cisco and/or its affiliates.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at:
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package commands
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func NewKvdbCommand(cli *AgentCli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "kvdb",
```

Comment:
This command returns `Failed to connect to Etcd: dial tcp: missing address` for all sub-commands. I do not see any flag where the address can be provided. Also, it should recognize default IP as other commands.